src/helpers/environment.sh

```diff
@@ -16,6 +17,17 @@ function ensure_git_repository {
 }
 
 
+function gittown_dev_version {
+  local base_dir=$( dirname "${BASH_SOURCE[0]}" )
+  local git_hash=$(git -C "$base_dir" rev-parse --short HEAD)
+  local git_date=$(git -C "$base_dir" --no-pager show -s --format=%cD HEAD | cut -d ' ' -f2-4)
+
+  if ! git -C "$base_dir" remote -v > /dev/null 2>&1 | grep "https://github.com/Homebrew/homebrew.git (fetch)"; then
```

Comment:
Why should it be in a subshell? The output redirection is in case the user isn't using Git 1.8.5 and above and the [`-C`](https://github.com/git/git/blob/5fd09df3937f54c5cfda4f1087f5d99433cce527/Documentation/RelNotes/1.8.5.txt#L115-L116) flag isn't supported.

Related to that, have we been considering the version of Git that users will have?